dts/bindings/espi/microchip,xec-espi-saf-v2.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 Intel Corporation
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip ESPI SAF version 2 controller
+
+compatible: "microchip,xec-espi-saf-v2"
+
+include: espi-controller.yaml
+
+properties:
+    reg:
+      description: mmio register space
+      required: true
+
+    girqs:
+      type: array
+      required: true
+      description: Array of encoded interrupt information
```

Comment:
I'm not familiar with this area, but why is this in `girqs` instead of `interrupts`?

@albertofloyd is this a standard property for espi devicetree bindings? Can you point me to any references in upstream linux bindings or another similarly authoritative place to make it clearer what is going on?

Reply:
the girqs are aggregated interrupts from the mec which hook to individual NVIC. The driver for a peripheral needs to know what interrupt in the girq it is hooked to clear the interrupt status in the girq.

Reply:
@mbolivar-nordic these are mchp interrupt specifics, not eSPI related